Maya: Set correct framerange with handles on file opening (#4664)
ynput/OpenPype: website/docs/admin_hosts_maya.md

This is useful to fix some specific renderer glitches and advanced hacking of Maya Scene files. `Patch name` is label for patch for easier orientation.
`Patch regex` is regex used to find line in file, after `Patch line` string is inserted. Note that you need to add line ending.

## Include/exclude handles by task type

> You can include or exclude handles globaly or by task type.

Reviewer: ```suggestion
You can include or exclude handles globally or by task type.

This setting is used to define the default frame range for new scenes or when using the **OpenPype > Set frame range** setting.
```

This made me think - does this actually also influence the **Validate Frame Range** validations or whatever it's called where it validates during publish whether an instance matches the asset document's frame range + handles?

Author: Yes it does influence the Validate Frame Range validator. And the "Repair" button when the validator fails repair correctly the framerange and the handles in the failed instances.

> The "Include handles by default" toggle is here to select weither you want handles or not in all task type. You can also chose in what task type you want handles or not.

Reviewer: ```suggestion
The "Include handles by default" defines whether by default handles are included or not for all task type. Additionally you can add a per task type override whether you want to include handles or not.
```

> The "Include handles by default" defines wether by default handles are included or not for all task type. Additionally you can add a per task type override wether you want to include handles or not.

Reviewer 2: ```suggestion
The "Include handles by default" defines whether by default handles are included. Additionally you can add a per task type override whether you want to include or exclude handles.
```

> ## Include/exclude handles by task type
> You can include or exclude handles globaly or by task type.
> 
> The "Include handles by default" toggle is here to select weither you want handles or not in all task type. You can also chose in what task type you want handles or not.
> 
> For example, in this image you can see that handles are included by default in all task types, except for the "Lighting" task, where the toggle is disabled.
> ![Include/exclude handles](assets/maya-admin_exclude_handles.png)
> 
> And here you can see thant handles are disabled by default, except in Animation task where it's enabled.
> ![Custom menu definition](assets/maya-admin_include_handles.png)
> 
> 

Reviewer 2: ```suggestion
## Include/exclude handles by task type
You can include or exclude handles, globally or by task type.

The "Include handles by default" toggle is here to select whether you want handles or not in all task type. You can also choose in which task types you want handles included.

For example, in this image you can see that handles are included by default in all task types, except for the `Lighting` task, where the toggle is disabled.
![Include/exclude handles](assets/maya-admin_exclude_handles.png)

And here you can see than the handles are disabled by default, except in `Animation` task where it's enabled.
![Custom menu definition](assets/maya-admin_include_handles.png)
```
